packages/cactus-plugin-ledger-connector-fabric/tsconfig.json

```diff
@@ -13,7 +13,10 @@
     "src/**/*.json"
   ],
   "exclude": [
-    "./src/test/typescript/fixtures/go/basic-asset-transfer/chaincode-typescript/**/*.ts"
+    "./src/test/typescript/fixtures/go/basic-asset-transfer/chaincode-typescript/**/*.ts",
+    "./src/test/typescript/fixtures/go/lock/chaincode-typescript/src/*.ts",
+    "./src/test/typescript/fixtures/go/lock-asset/chaincode-typescript/src/*.ts",
+    "./src/test/typescript/integration/fabric-v2-2-x/deploy-lock.test.ts"
```

Comment:
@jscode017 Why exclude a test case from compilation?

Reply:
Sorry, forget to clean up